src/server/auth/server/testing/auth_test.go

```diff
@@ -253,7 +253,12 @@ func getPachClientP(tb testing.TB, subject string, checkConfig bool) *client.API
 		if len(getAdminsResp.Admins) == 0 {
 			panic("it should not be possible to leave a cluster with no admins")
 		}
-		adminClient = getPachClientInternal(tb, getAdminsResp.Admins[0])
+		var currentAdmin string
+		for a := range getAdminsResp.Admins {
+			currentAdmin = a
+			break
+		}
+		adminClient = getPachClientInternal(tb, currentAdmin)
```

Comment:
Just trying to understand this block of code: I think if `getAdminsResp` is empty, then currentAdmin is `""` and `adminClient` is an anonymous client (vs `Admins[0]`, which crashes). Does that sound right? Is that expected behavior?